src/SamsungMdcConfigObject.cs

```diff
@@ -60,4 +65,13 @@ public class CustomInput
         [JsonProperty("inputIdentifier")]
         public string InputIdentifier { get; set; }
     }
-}
\ No newline at end of file
+
+    public class ActiveInputs : IKeyName
```

Comment:
@cdenig the reason the 3-series build is failing is that the syntax of assigning the default value to string.Empty on lines 72 and 75 isn't valid in .NET 3.5 CF.

This file is shared between the 3-series and 4-series solution.

To resolve, just leave a {get; set;} but add a constructor for the ActiveInputs class that sets the value of each to string.Empty if you need (though I would question if we really want to initialize the value to string.Empty as a default.